java-client/src/main/java/co/elastic/clients/elasticsearch/_helpers/bulk/BulkIngester.java

```diff
@@ -424,7 +429,8 @@ public Builder<Context> client(ElasticsearchClient client) {
         }
 
         /**
-         * Sets when to flush a new bulk request based on the number of operations currently added. Defaults to
+         * Sets when to flush a new bulk request based on the number of operations currently added.
+         * Defaults to
```

Comment:
Auto-formatting on save should be avoided to avoid mixing changes to the code and reformats, which the IDE got wrong here 😉